draft-ietf-rtcweb-jsep.xml

```diff
@@ -1433,18 +1490,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>The next step is to generate m= sections, as specified in
           <xref target="RFC4566"/> Section 5.14, for each
-          MediaStreamTrack that has been added to the PeerConnection via the
-          addStream method. (Note that this method takes a MediaStream, which
-          can contain multiple MediaStreamTracks, and therefore multiple m=
-          sections can be generated even if addStream is only called once.)
-          m=sections MUST be sorted first by the order in which the
-          MediaStreams were added to the PeerConnection, and
-          then by the alphabetical ordering of the media type for the MediaStreamTrack.
-          For example, if a MediaStream containing both an audio and a video
```

Comment:
The alphabetical thing is no longer needed (since we are track-based instead of stream-based).  The "order added" is retained elsewhere in the doc.